src/coreclr/vm/clrtocomcall.cpp

```diff
@@ -937,76 +937,3 @@ BOOL CLRToCOMMethodFrame::TraceFrame_Impl(Thread *thread, BOOL fromPatch,
 
     return TRUE;
 }
-
-#ifdef TARGET_X86
-
-#ifndef DACCESS_COMPILE
-
-CrstStatic   CLRToCOMCall::s_RetThunkCacheCrst;
-SHash<CLRToCOMCall::RetThunkSHashTraits> *CLRToCOMCall::s_pRetThunkCache = NULL;
-
-// One time init.
-void CLRToCOMCall::Init()
-{
-    CONTRACTL
-    {
-        THROWS;
-        GC_NOTRIGGER;
-        MODE_ANY;
-    }
-    CONTRACTL_END;
-
-    s_RetThunkCacheCrst.Init(CrstRetThunkCache);
```

Comment:
The crst level can be removed from `inc\CrstTypes.def` and generated `crsttypes_generated.h`.

Reply:
Good catch